modules/caddypki/command.go

```diff
@@ -53,19 +52,17 @@ This command will attempt to connect to Caddy's admin API running at
 '` + caddy.DefaultAdminListen + `' to fetch the root certificate. You may
 explicitly specify the --address, or use the --config flag to load
 the admin address from your config, if not using the default.`,
-		Flags: func() *flag.FlagSet {
-			fs := flag.NewFlagSet("trust", flag.ExitOnError)
-			fs.String("ca", "", "The ID of the CA to trust (defaults to 'local')")
-			fs.String("address", "", "Address of the administration API listener (if --config is not used)")
-			fs.String("config", "", "Configuration file (if --address is not used)")
-			fs.String("adapter", "", "Name of config adapter to apply (if --config is used)")
-			return fs
-		}(),
+		CobraFunc: func(cmd *cobra.Command) {
+			cmd.Flags().StringP("ca", "i", "", "The ID of the CA to trust (defaults to 'local')")
```

Comment:
```suggestion
			cmd.Flags().StringP("ca", "", "", "The ID of the CA to trust (defaults to 'local')")
```

The long form of this flag is two letters. :) (and the short form doesn't make as much sense IMO)